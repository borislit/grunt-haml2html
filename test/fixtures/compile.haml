- entries = [{  :title => "The Fellowship of the Ring",
                :body => "We should walk instead of fly" },
             {  :title => "The Two Towers",
                :body => "We're working on a third" },
             {  :title => "The Return of the King", 
                :body => "He's back from vacation" }]

!!! 5
%html
  %head
    %title Blog
    %meta{"http-equiv" => "Content-Type", :content => "text/html; charset=utf-8"}
    %link{"rel" => "stylesheet", "href" => "main.css", "type" => "text/css"}
  %body
    #header
      %h1 Blog
      %h2 Jason's Blog
    #content
      - entries.each do |entry|
        .entry
          %h3.title= entry[:title]
          %p.body= entry[:body]
    #footer
      %p
        All content copyright Â© Jason
        