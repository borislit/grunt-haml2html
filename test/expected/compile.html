<!DOCTYPE html>
<html>
  <head>
    <title>Blog</title>
    <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
    <link href='main.css' rel='stylesheet' type='text/css'>
  </head>
  <body>
    <div id='header'>
      <h1>Blog</h1>
      <h2>Jason's Blog</h2>
    </div>
    <div id='content'>
      <div class='entry'>
        <h3 class='title'>The Fellowship of the Ring</h3>
        <p class='body'>We should walk instead of fly</p>
      </div>
      <div class='entry'>
        <h3 class='title'>The Two Towers</h3>
        <p class='body'>We're working on a third</p>
      </div>
      <div class='entry'>
        <h3 class='title'>The Return of the King</h3>
        <p class='body'>He's back from vacation</p>
      </div>
    </div>
    <div id='footer'>
      <p>
        All content copyright Â© Jason
      </p>
    </div>
  </body>
</html>
